ArjunaJTS/jtax/tests/classes/com/hp/mwtests/ts/jta/jts/recovery/ImportedTransactionRecoveryUnitTest.java

```diff
@@ -168,6 +172,22 @@ public void commit(Xid xid, boolean onePhase) throws XAException {
 
         assertNotEquals("XAResources should be enlisted with different xids", xar1.getXid(), xar2.getXid());
 
+        // xid format verification of node names
+        assertEquals("Expecting the original xid has redefined node name",
```

Comment:
Did you mean 
assertEquals("Not expecting the original xid has redefined node name" ..
if not can you add some text to explain how the test works

Reply:
right, my wrong